### PR TITLE
Feat/patient creation

### PR DESCRIPTION
**Pull request by YanLabCHU**
Depends on: 
- https://github.com/Ferlab-Ste-Justine/clin-fhir-server/pull/32
- https://github.com/Ferlab-Ste-Justine/clin-fhir/pull/22

### CODE REVIEW

File: client/src/helpers/fhir/api/CreatePatient.ts [hunk 1]
```diff
@@ -42,3 +61,111 @@ export const createPatient = async (patient: Patient, familyGroup: FamilyGroup)
...
 };
+
+type CreatePatientFetusResponse = {
+  patient: Patient;
+  patientFetus: Patient;
+  familyGroup?: FamilyGroup;
+}
+
+export const createPatientFetus = async (patient: Patient) : Promise<CreatePatientFetusResponse> => {
+  const patientFetus = JSON.parse(JSON.stringify(patient)) as Patient;
```
Comment (francisprovost) [inline client/src/helpers/fhir/api/CreatePatient.ts:72]: Par curiosité ça change quoi de stringify + parse?

**Review by francisprovost — APPROVED** [review]
looks good

File: client/src/components/screens/PatientCreation/components/FormModal/index.tsx [hunk 2]
```diff
@@ -10,13 +10,12 @@ import { FormItemProps } from 'antd/lib/form';
 import { useForm } from 'antd/lib/form/Form';
 import moment from 'moment';
 import { RadioChangeEvent } from 'antd/lib/radio';
-import { set } from 'lodash';
+import { get, set } from 'lodash';
```
Comment (francisl) [inline client/src/components/screens/PatientCreation/components/FormModal/index.tsx:13]: using lodash that way, it make the bundle size explode. In CQDG and KidsFirst this pattern is proscribe.

Instead use
```javascript
import get from 'lodash/get';
import set from 'lodash/set';
```

A little bit verbose, but if use over the app you'll save around 100k.

File: client/src/helpers/fhir/api/CreatePatient.ts [hunk 3]
```diff
@@ -1,16 +1,28 @@
-import { get } from 'lodash';
+import { get, set } from 'lodash';
```
Comment (francisl) [inline client/src/helpers/fhir/api/CreatePatient.ts:1]: ☝️

File: client/src/components/screens/PatientCreation/components/FormModal/index.tsx [hunk 4]
```diff
@@ -101,6 +100,22 @@ function validateForm(formValues: any) {
...
+  hospital: string;
+}
+
+const extractMrnData = (patient: Patient) : MrnData | undefined => {
+  const identifier = patient.identifier.find((id) => get(id, 'type.coding[0].code') === 'MR');
+  if (identifier == null) {
+    return undefined;
+  }
+  return {
+    mrn: identifier.value,
```
Comment (francisl) [inline client/src/components/screens/PatientCreation/components/FormModal/index.tsx:114]: does your linter force you to do that? undefined is default if you only return your object on condition negation.

File: client/src/components/screens/PatientCreation/components/FormModal/index.tsx (hunk 2 above)
Comment (YanLabCHU) [reply client/src/components/screens/PatientCreation/components/FormModal/index.tsx:13]: Ok, I'll create a new PR with the changes

File: client/src/components/screens/PatientCreation/components/FormModal/index.tsx (hunk 4 above)
Comment (YanLabCHU) [reply client/src/components/screens/PatientCreation/components/FormModal/index.tsx:114]: I just prefer to be explicit about what the function returns

**Pull request merged**